frontend/lib/requests/galleries.ts

```
@@ -0,0 +1,47 @@
+import { Droplet } from "@/types";
+
+/**
+ * Gets the first 25 enrollments matching the specified criteria, unless overridden by `options`.
+ * @param options Strapi query modifiers.
+ * @returns The matching Droplets.
+ */
+export async function getGalleryBySlug(
+  slug: string,
+  {
+    sort,
+    filters,
```

ESLint

[error] 'filters' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)